### PR TITLE
Sdcb 9998 [Appium 2.0] JS Server Side

### PR DESCRIPTION
**Pull request by eryk-tkaczewski**

### CODE REVIEW

**Review by michal-przytarski — COMMENTED** [review]
You missed [Nightwatch sample](https://github.com/bitbar/test-samples/blob/master/samples/testing-frameworks/appium/server-side/javascript/README.md?plain=1#L4)  

It doesn't work on Appium1 on cloud on Android
https://cloud.bitbar.com/#testing/projects

File: samples/testing-frameworks/appium/client-side/javascript/nightwatch_web_example/nightwatch.json [hunk 1]
```diff
@@ -58,10 +58,12 @@
...
                 "browserName": "safari",
-                "appium:autoAcceptAlerts": "true",
-                "appium:newCommandTimeout":"120",
-                "appium:deviceName": "iPhone",
-                "appium:automationName": "XCUITest"
+                "appium:options": {
+                "automationName": "XCUITest",
+                "deviceName": "Iphone",
+                "newCommandTimeout": "120",
+                "autoAcceptAlerts": "true"
```
Comment (piotrsapiejewskismartbear) [inline samples/testing-frameworks/appium/client-side/javascript/nightwatch_web_example/nightwatch.json:65]: missing indent

File: samples/testing-frameworks/appium/client-side/javascript/nightwatch_web_example/nightwatch.json [hunk 2]
```diff
@@ -74,7 +76,12 @@
             },
             "capabilities": {
                 "platformName": "Android",
-                "browserName": "chrome"
+                "browserName": "chrome",
+                "appium:options": {
+                "deviceName": "Android device",
+                "automationName": "UiAutomator2",
+                "newCommandTimeout": 240
```
Comment (piotrsapiejewskismartbear) [inline samples/testing-frameworks/appium/client-side/javascript/nightwatch_web_example/nightwatch.json:83]: missing indent

File: samples/testing-frameworks/appium/server-side/javascript/biometricInstrumentation/webdriver.io/android.wdio.conf.js [hunk 3]
```diff
@@ -40,11 +40,12 @@ exports.config = {
...
-
-      "appium:deviceName": "Android device",
-      "appium:automationName": "UiAutomator2",
-      "appium:app": path.resolve("application.apk"),
-      "appium:newCommandTimeout": 240,
+      'appium:options': {
+        'deviceName': 'Android device',
+        'automationName': 'UiAutomator2',
+        'app': path.resolve('application.apk'),
+        'newCommandTimeout': 240,
```
Comment (piotrsapiejewskismartbear) [inline samples/testing-frameworks/appium/server-side/javascript/biometricInstrumentation/webdriver.io/android.wdio.conf.js:47]: keys don't need single-quote wrappers if they're simple words

File: samples/testing-frameworks/appium/server-side/javascript/biometricInstrumentation/webdriver.io/ios.wdio.conf.js [hunk 4]
```diff
@@ -40,12 +40,13 @@ exports.config = {
...
-      "appium:deviceName": "Local Device",
-      "appium:automationName": "XCUITest",
-      "appium:app": path.resolve("application.ipa"),
-      "appium:newCommandTimeout": 240,
+      'appium:options': {
+        'udid': process.env.IOS_UDID,
+        'deviceName': 'Local Device',
+        'automationName': 'XCUITest',
+        'app': path.resolve('application.ipa'),
+        'newCommandTimeout': 240,
```
Comment (piotrsapiejewskismartbear) [inline samples/testing-frameworks/appium/server-side/javascript/biometricInstrumentation/webdriver.io/ios.wdio.conf.js:48]: keys don't need single-quote wrappers if they're simple words

File: samples/testing-frameworks/appium/server-side/javascript/webdriver.io/android.conf.js [hunk 5]
```diff
@@ -46,13 +46,12 @@ exports.config = {
...
-        'appium:automationName': 'UiAutomator2',
-        'appium:app': path.resolve('application.apk'),
-        'appium:appActivity': '.BitbarSampleApplicationActivity',
-        'appium:appPackage': 'com.bitbar.testdroid',
-        'appium:newCommandTimeout': 240
+        'appium:options': {
+            'deviceName': 'Android device',
+            'automationName': 'UiAutomator2',
+            'app': path.resolve('application.apk'),
+            'newCommandTimeout': 240
```
Comment (piotrsapiejewskismartbear) [inline samples/testing-frameworks/appium/server-side/javascript/webdriver.io/android.conf.js:53]: keys don't need single-quote wrappers if they're simple words

File: samples/testing-frameworks/appium/server-side/javascript/webdriver.io/ios.conf.js [hunk 6]
```diff
@@ -40,12 +40,12 @@ exports.config = {
...
-        'appium:udid': process.env.IOS_UDID,
-        'appium:deviceName': 'Local Device',
-        'appium:automationName': 'XCUITest',
-        'appium:app': path.resolve('application.ipa'),
-        'appium:newCommandTimeout': 240,
+        'appium:options' : {
+        'udid': process.env.IOS_UDID,
+        'automationName': 'XCUITest',
+        'app': path.resolve('application.ipa'),
+        'newCommandTimeout': 240,
```
Comment (piotrsapiejewskismartbear) [inline samples/testing-frameworks/appium/server-side/javascript/webdriver.io/ios.conf.js:47]: missing indent
keys don't need single-quote wrappers if they're simple words

**Pull request merged**